201-sql-auditing-server-policy-to-blob-storage/azuredeploy.json

```diff
@@ -34,9 +34,19 @@
       "metadata": {
         "description": "The name of the auditing storage account."
       }
+    },
+    "isStorageBehindVnet": {
+      "type": "bool",
+      "defaultValue": false,
+      "metadata": {
+        "description": "Enable Auditing to storage behind Virtual Network or firewall rules. the user deploying the template must have an administrator or owner permissions."
+      }
     }
-  },
-  "variables": {},
+	},
+  "variables": {
+		"StorageBlobContributor": "[subscriptionResourceId('Microsoft.Authorization/roleDefinitions', 'ba92f5b4-2d11-453d-a403-e96b0029c9fe')]",
+		"uniqueRoleGuid": "[guid(resourceId('Microsoft.Storage/storageAccounts',  parameters('storageAccountName')), variables('storageBlobContributor'),     resourceId('Microsoft.Sql/servers', parameters('sqlServerName')))]"
```

Comment:
add variables('StorageBlobContributor') to your seed

Reply:
This is a pre defined Guid for StorageBlobContributor role.